### PR TITLE
Put chunkhash in build paths

### PR DESCRIPTION
**Pull request by morgs32**

### CODE REVIEW

File: packages/react-scripts/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@brickwork-software/react-scripts",
-  "version": "1.3.1",
+  "version": "1.3.2",
```
Comment (wilsonyhlee) [inline packages/react-scripts/package.json]: should this be 1.3.3?

**Review by wilsonyhlee — COMMENTED** [review]
Left a question

File: packages/react-scripts/package.json (hunk 1 above)
Comment (morgs32) [reply packages/react-scripts/package.json]: No... what makes you think that?

**Review by wilsonyhlee — APPROVED** [review]
👍

**Pull request merged**